php-checks/src/main/java/org/sonar/php/checks/LocalVariableAndParameterNameCheck.java

```diff
@@ -141,12 +139,7 @@ private void reportIssue(String type, Tree tree, String varName) {
   }
 
   private static ExpressionTree getLeftHandExpression(Tree assignmentExpr) {
-    ExpressionTree leftExpression;
-    if (assignmentExpr.is(Kind.ASSIGNMENT_BY_REFERENCE)) {
-      leftExpression = ((AssignmentByReferenceTree) assignmentExpr).variable();
-    } else {
-      leftExpression = ((AssignmentExpressionTree) assignmentExpr).variable();
-    }
+    ExpressionTree leftExpression = ((AssignmentExpressionTree) assignmentExpr).variable();;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)